rt_gene/scripts/estimate_blink.py

```diff
@@ -76,6 +80,16 @@ def callback(self, msg):
             '\033[2K\033[1;32mTime now: {:.2f} message color: {:.2f} latency: {:.2f}s for {} subject(s) {:.0f}Hz\033[0m'.format(
                 (_now.to_sec()), timestamp.to_sec(), np.mean(self._latency_deque), len(subjects), np.mean(self._freq_deque)), end="\r")
 
+    def publish_msg(self, timestamp, subjects, probabilities):
+        blink_msg_list = MSG_BlinkList()
+        blink_msg_list.header.stamp = timestamp
+        blink_msg_list.header.frame_id = '0'
+        for subject_id, p in zip(subjects.keys(), probabilities):
+            blink_msg = MSG_Blink()
+            blink_msg.subject_id = str(subject_id)
+            blink_msg.blink = p >= self.threshold
+            blink_msg_list.subjects.append(blink_msg)
```

Comment:
Doesn't the actual publishing miss? This just seems to prepare the messages.

Reply:
Sorry i thought i ammended my commit to actually publish it. That's rather embarassing!